packages/devtools_app/lib/src/screens/memory/memory_heap_tree_view.dart

```diff
@@ -1462,10 +1468,10 @@ class _LibraryRefColumn extends TreeColumnData<Reference> {
     // Should never be empty when we're displaying in table.
     assert(!dataObject.isEmptyReference);
 
-    String value;
+    String? value;
```

Comment:
make non null and make getValue return type a non null String.

Reply:
done